geco/mips/facility_location.py

```diff
@@ -79,6 +72,12 @@ def capacitated_facility_location(n_customers, n_facilities, transportation_cost
 
 @py_random_state(3)
 def cornuejols_instance_params(n_customers, n_facilities, ratio, seed):
+    """This code is heavily based on the code available in
```

Comment:
This might become lengthy, but I think we should also mention the cornuejols paper here. While the code is based on the one mentioned here, the use of sampled x and y coordinates to determine costs etc. is from the later.

Reply:
Maybe we can adopt networkx reference part in the docstrings. See e.g. [here](https://github.com/networkx/networkx/blob/master/networkx/algorithms/coloring/greedy_coloring.py#L319).

Reply:
Sounds good

Reply:
But then we should create an issue to change other docstrings also

Reply:
I agree.